### PR TITLE
documentation/ZENKO-3062_edit_a_storage_location

### PR DESCRIPTION
**Pull request by wabernat**
Added sections for: add storage locations, add transient storage locations, edit storage locations, and delete storage locations for XDM UI, with graphic files.

Deleted/modified Orbit sections and indexes as they broke.

### CODE REVIEW

File: docs/docsource/operation/XDM_GUI/location_management/add_a_storage_location.rst [hunk 1]
```diff
@@ -1,33 +1,60 @@
 .. _Add a Storage Location:
 
 Add a Storage Location
-=========================
+======================
+
+Though you can learn by nominating arbirary Zenko endpoints in a local file
```
Comment (miniscruff) [inline docs/docsource/operation/XDM_GUI/location_management/add_a_storage_location.rst]: Typo?
```suggestion
Though you can learn by nominating arbitrary Zenko endpoints in a local file
```

File: docs/docsource/operation/XDM_GUI/location_management/add_a_storage_location.rst [hunk 2]
```diff
@@ -1,33 +1,60 @@
...
 To add a storage location:
 
-#. From the Account view, select an account name to expose account information.
+#. From the **Accounts** view, select an account name to expose account
+   information.
 
    .. image:: ../../Graphics/xdm_ui_account_view.png
+      :width: 100%
+
+
```
Comment (miniscruff) [inline docs/docsource/operation/XDM_GUI/location_management/add_a_storage_location.rst]: I would remove a newline
```suggestion

```

File: docs/docsource/operation/XDM_GUI/location_management/add_a_storage_location.rst (hunk 1 above)
Comment (wabernat) [reply docs/docsource/operation/XDM_GUI/location_management/add_a_storage_location.rst]: You, sir, have a keen eye!

**Review by miniscruff — APPROVED** [review]

**Review by lucieleonard — APPROVED** [review]

**Review by saitharunthodupunuru — APPROVED** [review]

**Pull request merged**